projects/igniteui-angular/package.json

```diff
@@ -77,14 +77,14 @@
         "@igniteui/material-icons-extended": "^2.4.0"
     },
     "peerDependencies": {
-        "@angular/common": "^11.0.0",
-        "@angular/core": "^11.0.0",
-        "@angular/animations": "^11.0.0",
-        "@angular/forms": "^11.0.0",
+        "@angular/common": "^11.2.0",
+        "@angular/core": "^11.2.0",
+        "@angular/animations": "^11.2.0",
+        "@angular/forms": "^11.2.0",
         "web-animations-js": "^2.3.2"
     },
     "igxDevDependencies": {
-        "@igniteui/angular-schematics": "~11.0.700"
+        "@igniteui/angular-schematics": "~11.1.710-beta.0"
```

Comment:
Should this really be a dependency on a beta version?

Reply:
Yes, because will release a beta for the Ignite UI CLI in order to test it and the schematics.